index.bs

```diff
@@ -120,10 +120,10 @@ with the <{a}> element.
 
 The <dfn for="a" element-attr>attributionsourceeventid</dfn> attribute is a string
 containing information about the `attribution source` and will be supplied in the
-[=attribution report=].
+[=event-level attribution report=].
```

Comment:
While we're making this change, what do we think about removing `attribution` from the name, since we're already in the attribution reporting spec?

Reply:
That makes sense to me, but we have `attribution` in many other structures as well...

Reply:
We can, but now seems like a good time to start :)